oversight/management/commands/sensordaemon.py

```diff
@@ -1,10 +1,10 @@
 import logging
 import threading
-import Queue
-from SimpleXMLRPCServer import SimpleXMLRPCServer
+from multiprocessing import Queue
```

Comment:
das direkte äquivalent zu `Queue.Queue` in python2 wäre `queue.Queue`: https://docs.python.org/3.8/library/queue.html

Reply:
Wird in dem Fall queue.task_done() überhaupt benötigt? Ich sehe kein queue.join, oder ähnliches.

Reply:
Stimmt wahrscheinlich; ich würde es allerdings lassen fallst den code mal umbaust (sauberer ist es auf jeden fall einen task zu quittieren)